docs/src/main/sphinx/security/cert-auth.rst

```diff
@@ -0,0 +1,108 @@
+==========================
+Certificate authentication
+==========================
+
+Trino can be configured to support certified clients seeking validation from a
+Trino server. This authentication method is only provided to support sites that
+have an absolute requirement for client authentication *and already have* client
+certificates for each client. Sites in this category have an existing PKI
+infrastructure, possibly including an onsite CA.
+
+This feature is *not* appropriate for sites that would need to generate a set of
```

Comment:
I don't think we need to emphasize "not"

Reply:
Again, we're trying to steer users away from this page. Only 1 in 500 sites will need it.

Reply:
In other places we use **strongly suggest** etc. With bold. I suggest we stick with the bold convention for any emphasis.

Reply:
I am in favour of leaving as is. "Strongly suggest" just feels too weak..